src/react/deckgl.js

```diff
@@ -78,6 +80,17 @@ export default class DeckGL extends React.Component {
     this._updateLayers(nextProps);
   }
 
+  /* Public API */
+  elementsInScope(topLeft, bottomRight) {
```

Comment:
To support the review, write up the doc pages for the public APIs, either as temp comment here or in the doc structure.